recipes/xorg/all/conanfile.py

```diff
@@ -47,7 +47,8 @@ def system_requirements(self):
                 packages = ["Xorg-x11-devel"]
             else:
                 self.warn("Do not know how to install 'xorg' for {}.".format(tools.os_info.linux_distro))
-            package_tool.install(update=True, packages=packages)
+            for p in packages:
+                package_tool.install(update=True, packages=p)
```

Comment:
You can install all together. Installer each package individually will take more time.

Reply:
not according to https://docs.conan.io/en/latest/reference/conanfile/methods.html#systempackagetool:
```
If packages is a list the first available package will be picked (short-circuit like logical or). Note: This list of packages is intended for providing alternative names for the same package, to account for small variations of the name for the same package in different distros. To install different packages, one call to install() per package is necessary.
```

Reply:
or I misunderstood you, and you are actually suggesting to do a single `package_tool.install(update=True, packages=" ".join(packages))` ?

Reply:
@ericLemanissier I changed it months ago: https://github.com/conan-io/conan/blob/63b6a28ab36706144dbdc0ddf40ead09522fecc1/conans/client/tools/system_pm.py#L103. However, I forgot to update that specific clause in docs: "(short-circuit like logical or)" ... I'll fix that.